packages/loopring_v3/contracts/aux/bridge/BatchDepositor.sol

```diff
@@ -287,21 +288,21 @@ abstract contract BatchDepositor is IBatchDepositor, ReentrancyGuard
         }
     }
 
-    function _hashTransfers(bytes memory transfersData)
+    function _hashTransferBatch(uint batchID, bytes memory transfersData)
         internal
         pure
         returns (bytes32)
     {
-        return keccak256(transfersData);
+        return keccak256(abi.encodePacked(batchID, transfersData));
```

Comment:
I did the same in the beginning, but I changed it to keep the ID separate to save gas and keep things simple. `abi.encodePacked` isn't a cheap operation in most cases, especially when using it on data already in memory. So I would prefer it's not done this way because it increases gas costs.

Reply:
Thank you, will revert this change then.